registry/README.md

```diff
@@ -1,32 +1,32 @@
-# netdata registry
+# Netdata registry
 
 Netdata registry implements the `my-netdata` menu on netdata dashboards.
 The `my-netdata` menu lists the netdata servers you have visited.
 
 ## Why?
 
-netdata provides distributed monitoring.
+Netdata provides distributed monitoring.
 
 Traditional monitoring solutions centralize all the data to provide unified dashboards across all servers. Before netdata, this was the standard practice. However it has a few issues:
 
 1. due to the resources required, the number of metrics collected is limited.
 2. for the same reason, the data collection frequency is not that high, at best it will be once every 10 or 15 seconds, at worst every 5 or 10 mins.
-2. the central monitoring solution needs dedicated resources, thus becoming "another bottleneck" in the whole ecosystem. It also requires maintenance, administration, etc.
-3. most centralized monitoring solutions are usually only good for presenting *statistics of past performance* (i.e. cannot be used for real-time performance troubleshooting).
+3. the central monitoring solution needs dedicated resources, thus becoming "another bottleneck" in the whole ecosystem. It also requires maintenance, administration, etc.
```

Comment:
You can use:
```
1.
1.
1.
```
for lists. This will be renederd as a numbered list but eases a way to expand them in the future.

Reply:
Nice tip, thanks!